check-tldr-fix/check-tldr-fix.ipynb

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
<p>Yes, it does!</p>

 _Reply via <a href='https://app.reviewnb.com/slomeks/oku_notebooks/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>